nevergrad/functions/mixsimulator/core.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Based on https://github.com/Foloso/MixSimulator/tree/nevergrad_experiment
+
+import numpy as np
+from ..base import ExperimentFunction
+from mixsimulator.MixSimulator import MixSimulator  #type: ignore
+
+
+class OptimizeMix(ExperimentFunction):
+    """
+        MixSimulator is an application with an optimization model for calculating 
+        and simulating the least cost of an energy mix under certain constraints.
+        
+        For now, it uses a default dataset (more will be added soon).
+        
+        For more information, visit : https://github.com/Foloso/MixSimulator     
+        
+        Parameters
+        ----------
+        time: int
+            total time over which it evaluates the mix (must be in hour)
+                
+    """
+    def __init__(self,time: int = 168) -> None:
+        self.__mix = MixSimulator()
```

Comment:
I would not recommend using a double underscore, this has complicated behaviors when it comes to subclassing (the name ends up being different). Up to you if you are confortable with it though, on my side I tend to go always with a single underscore

Reply:
It's true that it can a problem for scalability.. Gonna fix that now too